packages/valory/skills/decision_maker_abci/behaviours/decision_receive.py

```diff
@@ -221,20 +223,81 @@ def _get_decision(
 
         return self.mech_response.result.vote, self.mech_response.result.confidence
 
-    def _is_profitable(self, confidence: float) -> bool:
+    def _calc_binary_shares(self, net_bet_amount: int, vote: int) -> Tuple[int, int]:
+        """Calculate the claimed shares. This calculation only works for binary markets."""
```

Comment:
The output of this method was tested with multiple values and the returned number of shares is accurate.